home-page

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Binabox NFT Collection</title>
+</head>
+<body>
+
+    <h2>Binabox</h2>
+    <p><a href="#">Home</a></p>
+    <p><a href="#">About</a></p>
+    <p><a href="#">Roadmap</a></p>
+    <p><a href="#">Collection</a></p>
+    <p><a href="#">FAQs</a></p>
+    <p><a href="#">Pages</a></p>
+
+    <br>
```

Comment:
I would recommend removing this tag `<br>`  for now and adding it in the future only if absolutely necessary. We will manage the various indents later using CSS and most likely we will not need this tag